File: imc/IMC.java
// Precisa importar java.util.Scanner para ler entrada do teclado.
// Cada variável tem tipo declarado explicitamente (double, String).
// printf() formata casas decimais, no lugar das f-strings do Python.

import java.util.Scanner;

public class IMC {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite seu peso (kg): ");
        double peso = scanner.nextDouble();

        System.out.print("Digite sua altura (m): ");
        double altura = scanner.nextDouble();

        double imc = peso / (altura * altura);
        String classificacao;

        if (imc < 18.5) {
            classificacao = "Abaixo do peso";
        } else if (imc < 25) {
            classificacao = "Peso normal";
        } else if (imc < 30) {
            classificacao = "Sobrepeso";
        } else if (imc < 35) {
            classificacao = "Obesidade grau I";
        } else if (imc < 40) {
            classificacao = "Obesidade grau II";
        } else {
            classificacao = "Obesidade grau III";
        }

        System.out.printf("Seu IMC é %.2f - %s%n", imc, classificacao);

        scanner.close();
    }
}

File: imc/imc/imc.py
# input() retorna string; usamos float() para converter.
# Não é preciso declarar o tipo da variável (tipagem dinâmica).
# f-strings facilitam formatar números dentro do texto.

peso = float(input("Digite seu peso (kg): "))
altura = float(input("Digite sua altura (m): "))

imc = peso / (altura ** 2)

if imc < 18.5:
    classificacao = "Abaixo do peso"
elif imc < 25:
    classificacao = "Peso normal"
elif imc < 30:
    classificacao = "Sobrepeso"
elif imc < 35:
    classificacao = "Obesidade grau I"
elif imc < 40:
    classificacao = "Obesidade grau II"
else:
    classificacao = "Obesidade grau III"

print(f"Seu IMC é {imc:.2f} - {classificacao}")
